index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <main>
+      <header class="container">
+        <a href="#" class="logo"><img src="./Logo_placeholder.png" alt="Logo"/></a>
+        <nav>
+          <a href="#" class="menu"><img src="./Enabled.png" alt="Menu button"/></a>
+        </nav>
+      </header>
+      <section class="headline">
+        <h1 class="primary-text">Hey There. I'm Festus</h1>
+        <h2 class="supporting-text">I am a Software Developer</h2>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <ul class="social-media">
+          <li>
+            <a href="#" class="git"
+              ><img src="./Github.png" alt="Github logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="linkedin"
+              ><img src="./linkedIn.png" alt="Linkedin logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="angelist"
+              ><img src="./angelist.png" alt="Angelist logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="twitter"
+              ><img src="./Twitter.png" alt="Twitter logo"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="vector"
+              ><img src="./Vector.png" alt="Vector logo"
+            /></a>
+          </li>
+        </ul>
```

Comment:
- [ OPTIONAL] you've done a great job implementing the changes from the previous reviewer, but I would suggest fixing this icon links. 
![image](https://user-images.githubusercontent.com/43877689/176852735-0837bd29-6e92-44ce-ae76-cac2d80ddbe3.png)